pages/specification/json-hyper-schema/_index.md

```diff
@@ -13,6 +13,531 @@ In essence:
+    }
+  ]
+}
+```
+
+#### Multiple relations
+
+A multiple relations associating a comments collection with two different relationship types.
+
+```json
+{
+  "links": [
+    {
+      "href": "https://example.com/reviews",
+      "rel": ["collection", "https://example.com/relations/reviews"]
+    }
+  ]
+}
+```
+
+### Base URIs
+
+A base URI is used to resolve relative URIs.
+
+#### Default
+
+By default, the base URI for relative URIs in LDOs is the URI used to retrieve the resource
+
+```http
+GET https://example.com/books/1
+```
+
+```json
+{
+  "links": [
+    {
+      "href": "/author/1",
+      "rel": "author"
+    }
+  ]
+}
+```
+
+Link Target: `https://example.com/author/1`
+
+#### Setting the base URI
+
+`base` can be used to alter the base URI. It is a URI Template and can be relative.
+
+```http
+GET https://example.com/myapi/v3/books/1
+```
+
+```json
+{
+  "base": "/myapi/v3/",
+  "links": [
+    {
+      "href": "author/1",
+      "rel": "author"
+    }
+  ]
+}
+```
+
+Link Target: `https://example.com/myapi/v3/author/1`
+
+#### Setting a link's "anchor"
+
+A link is a connection between two resources. The source of the link is called the "anchor" and the destination is called the "target. The "anchor" usually doesn't need to be specified because it's understood to be the resource the link appears in. `anchor` allows you to change the link's "anchor" to something other than the resource it appears in.
+
+When `anchor` appears in an LDO, it becomes the base URI for resolving `href`.
+
+> NOTE: I don't have an example because I can't think of any reason someone would want to do this. It's an anti-pattern at best. It might be best to just leave `anchor` and `anchorPointer` undocumented.
+
+### URI Templates
+
+`href`, `base`, and `anchor` are URI Templates
+
+#### Template variables
+
+##### Link applicability
+
+```json
+{
+  "type": "object",
+  "properties": {
+    "documentation": {
+      "links": [
+        {
+          "href": "/docs",
+          "rel": "about"
+        }
+      ]
+    }
+  }
+}
+```
+
+```json
+{}
+```
+
+**about**: N/A
+
+> Note: Links are annotations, which means they're attached to a location in the JSON instance. If the location of the link doesn't exist in the JSON instance, the link doesn't apply.
+
+```json
+{
+  "documentation": true
+}
+```
+
+**about**: `https://example.com/docs`
+
+##### Required/Optional variables
+
+```json
+{
+  "links": [
+    {
+      "href": "/books?page={next}{&perPage}",
+      "rel": "next",
+      "templateRequired": ["next"]
+    },
+    {
+      "href": "/books?page={previous}{&perPage}",
+      "rel": "previous",
+      "templateRequired": ["previous"]
+    }
+  ]
+}
+```
+
+```json
+{
+  "list": [
+    { ... book 1 ... },
+    { ... book 2 ... },
+    { ... book 3 ... }
+  ],
+  "page": 0,
+  "next": 1
+}
+```
+
+**next**: `https://example.com/books?page=1`
+
+**previous**: N/A
+
+Note: `previous` doesn't apply because the required property "previous" is not present.
```

Comment:
See the comment above for notes and notices. 
thought: the reason why `previous` doesn't appear in this example is because the current page (0) doesn't have a previous page. Sating this would make the NOTE more specific and informative. 

suggestion: "The `previous` link is not defined in this JSON example because the current page (`page: 0`) had no preceding page to link to. However, in other instances, a `previous` link might be available with the "rel": "previous" property in the response."